tsconfig.json

```diff
@@ -14,6 +14,10 @@
       "node",
       "mocha",
       "chrome"
+    ],
+    "lib": [
+      "dom",
+      "es7"
```

Comment:
I tested here and isn't needed to add "es7", as the node env supports already the array.prototype.include.
Also, I think there's no need in adding dom, as the entire code runs over node (not considering the injected source).

Reply:
Strange, I get a lot of errors when I try to remove those...

ERROR in _/node_modules/@types/filewriter/index.d.ts
(10,29): error TS2304: Cannot find name 'EventTarget'.

error TS2339: Property 'includes' does not exist on type 'string | string[]'.

Reply:
Strange, TS runs fine here. This happens on the `yarn build`?
Can you confirm you're seeing: `ts-loader: Using typescript@2.3.3` on the log? Maybe is a different version or so (should not be, because of the yarn.lock)

Reply:
I see, for some reason you building process is running over `node_modules`. Are you using the `ts` directly?

Reply:
I'm sorry but I don't understand your question, I run npm run test as usual + when I try to push all the unit tests runs automatically, so I cant push my changes without those lines

Reply:
I was meaning not the tests but the building process. As the project is using yarn is also important you install the dependencies using `yarn install` so it can use the lockfile to get the exact version of each dependency.
After that try to run the build process with `yarn build`

Reply:
I ran yarn install, yarn build - the same - error TS2339: Property 'includes' does not exist.

Reply:
Something is wrong to make the ts-loader run over your node_modules in you environment. This shouldn't be happening :(
Can you give information about your environment, if this is happening because of the setup, is something which needs to be fixed.

Anyway, again this line are't necessary (you can check on the continuous integration tests).

Reply:
node -v = 8.11.1
npm v5.6.0
what else?

Reply:
I've just ran the build in a clean setup. No problems at all, the `ts-loader` is indeed excluding the `node_modules` on the webpack build (the same as in the continous integration builds).
Again, this lines aren't necessary, as can be fixing some local problem of your machine, but isn't happening in other envs.
Sorry if this is happening there, but can you remove this lines?